Common/ConfigParser.cs

```diff
@@ -127,6 +133,27 @@ public class UploaderSubsection
             public IncludeExcludePatterns AssemblyPatterns { get; set; }
         }
 
+        /// <summary>
+        /// Contains the thresholds for purging trace files from the upload archives.
+        /// </summary>
+        public class PurgeUploadArchivesSection
+        {
+            /// <summary>
+            /// Minimal timespan to keep uploaded traces.
+            /// </summary>
+            public TimeSpan UploadedTraces { get; set; } = TimeSpan.MinValue;

```

Comment:
After reading the rest of the code I understand that MinValue here causes the uploader to not perform a purge. But when I initially saw this line, I thought that the default is "clean immediately". Can we represent the default in a cleaner way somehow to make it clear to the reader here that the default is "do nothing" and not "clean up everything right away"?

Also, the documentation of TimeSpan.MinValue never mentions that this is a negative value, so even after reading that I didn't understand this default value. Since we already use TimeSpan.Zero to mean "clean up right away", how about using null to represent "do nothing" and documenting that in the comment above the fields?